README.md

```diff
@@ -55,8 +55,8 @@ stack build
 
 ## Calling Weeder
 
-To call Weeder, you first need to provide a configuration file, `weeder.dhall`. Weeder uses
-[Dhall](https://dhall-lang.org) as its configuration format, and configuration
+To call Weeder, you first need to provide a configuration file, `weeder.toml`. Weeder uses
+[TOML](https://toml.io/en/) as its configuration format, and configuration
```

Comment:
I think we should remove "and configuration files have the type...", as that's fairly Dhall specific.